src/scripts.js

```diff
@@ -1 +1,48 @@
-console.log("Hello World");
\ No newline at end of file
+//Query Selectors
+let userInfo = document.querySelector(".user-info");
+let displayMessage = document.querySelector(".display-message");
+let stepGoalComparison = document.querySelector(".step-goal-comparison");
+
+//Event Listeners
+window.addEventListener("load", initializeUserInfo);
+
+//Global Variables
+let activeUser = new User(sampleData[1]);
```

Comment:
I suspect `User`, `UserRepository`, and `sampleData` need to be defined here somewhere.

Reply:
I think eventually we will have to instantiate all the different classes and the data file here, as for now we are just using the smaller sample data size just to check we are accessing the information needed correctly. I think that is what you are referring to? Or were you talking about needing to define the class itself? Because i think that is in the HTML at the bottom in the script tags?